frame/system/src/lib.rs

```diff
@@ -269,12 +269,6 @@ pub mod pallet {
 				0
 			}
 		}
-
-		fn integrity_test() {
-			T::BlockWeights::get()
-				.validate()
-				.expect("The weights are invalid.");
-		}
```

Comment:
why do you remove it ?

In general I think ppl should review their PR and asking themself the question:
* is this change part of the PR ?
* is it just a not needed extra (in this case you can put a comment saying that it is an extra not really part of the PR, and the reason why it is done)